### PR TITLE
Don't use `oldtime` feature of chrono 

### PR DESCRIPTION
**Pull request by nibon7**
# Description
`chrono` crate enables `oldtime` feature by default, which has a vulnerability (https://rustsec.org/advisories/RUSTSEC-2020-0071). This PR tries to remove `time` v0.1.45 completely from nu and add an audit CI to check for security vulnerabilities.

:hand: Wait for the following PRs:
- [x] https://github.com/nushell/reedline/pull/599
- [x] https://github.com/bspeice/dtparse/pull/44
- [x] https://github.com/Byron/trash-rs/pull/75
- [x] https://gitlab.com/imp/chrono-humanize-rs/-/merge_requests/15

# User-Facing Changes

# Tests + Formatting

# After Submitting

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -165,7 +165,10 @@ bench = false
 # To use a development version of a dependency please use a global override here
 # changing versions in each sub-crate of the workspace is tedious
 [patch.crates-io]
-# reedline = { git = "https://github.com/nushell/reedline.git", branch = "main"}
+reedline = { git = "https://github.com/nushell/reedline.git", branch = "main" }
+dtparse = { git = "https://github.com/nibon7/dtparse.git", branch = "chrono" }
```
Comment (fdncred) [inline Cargo.toml:169]: These type of things are fine for now, but we can't publish releases unless the dependency crates are in crates.io.

**Comment by fdncred** [comment]
Thanks @nibon7 for trying to push forward on this proactively! I just wonder if these crates will show up in crates.io or not?

File: Cargo.toml (hunk 1 above)
Comment (nibon7) [reply Cargo.toml:169]: We're waiting for the upstream to merge related PRs and this is temporary to make to the audit CI works :)

Comment (fdncred) [reply Cargo.toml:169]: ok, i put checkboxes in the description so we can follow that status easier. Thanks!

**Comment by sholderbach** [comment]
Updated as `chrono-humanize` is now released with the fix.

**Comment by amtoine** [comment]
what is this PR missing? :yum:

**Comment by sholderbach** [comment]
> what is this PR missing? 😋

https://github.com/bspeice/dtparse/pull/44 landing and being released. Currently this is still only a patch which can not be released as a crate.

**Comment by amtoine** [comment]
> [bspeice/dtparse#44](https://github.com/bspeice/dtparse/pull/44) landing and being released. Currently this is still only a patch which can not be released as a crate.
>
yup makes sense, i just wasn't sure if the term *related* was that strong in the PR description above :relieved:

**Comment by fdncred** [comment]
finally, all the upstream prs landed. @nibon7 can you get the ci green again please?

**Review by sholderbach — APPROVED** [review]
Thanks!

**Pull request merged**